tests/Feature/QuizTest.php

```diff
@@ -85,41 +87,41 @@ public function test_addQuiz_malformedDescription(): void
 
         $response = $this->postJson('/api/quizzes', $testData);
         $response->assertStatus(422)
-        ->assertJson(function(AssertableJson $json) {
-            $json->hasAll(['message', 'errors'])
-            ->has('errors', function (AssertableJson $json) {
-                $json->hasAll( 'description');
+            ->assertJson(function (AssertableJson $json) {
+                $json->hasAll(['message', 'errors'])
+                    ->has('errors', function (AssertableJson $json) {
+                        $json->hasAll('description');
+                    });
             });
-        });
     }
 
     public function test_getSingleQuiz_success(): void
     {
-        Answer::factory()->count(5)->create();
+        Answer::factory()->count(1)->create();
```

Comment:
if you're changing it to `count(1)` then why have the `count()` at all? It makes 1 without including it.

It might be interesting to create 3 and then get index 3 tho.